### ISSUE TITLE
LocationGMS.fireComplete(LocationGMS.java:210)

### ISSUE THREAD

**Issue by JustinLim0318** [OneSignal/OneSignal-Android-SDK#233]
```Java
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'void java.lang.Thread.interrupt()' on a null object reference
       at com.onesignal.LocationGMS.fireComplete(LocationGMS.java:210)
       at com.onesignal.LocationGMS.receivedLocationPoint(LocationGMS.java:233)
       at com.onesignal.LocationGMS.access$200(LocationGMS.java:54)
       at com.onesignal.LocationGMS$GoogleApiClientListener.onConnected(LocationGMS.java:247)
       at com.google.android.gms.common.internal.zzm.zzq(Unknown Source)
       at com.google.android.gms.internal.zzaal.zzo(Unknown Source)
       at com.google.android.gms.internal.zzaaj.zzvE(Unknown Source)
       at com.google.android.gms.internal.zzaaj.onConnected(Unknown Source)
       at com.google.android.gms.internal.zzaan.onConnected(Unknown Source)
       at com.google.android.gms.internal.zzzy.onConnected(Unknown Source)
       at com.google.android.gms.common.internal.zzl$1.onConnected(Unknown Source)
       at com.google.android.gms.common.internal.zzf$zzj.zzwZ(Unknown Source)
       at com.google.android.gms.common.internal.zzf$zza.zzc(Unknown Source)
       at com.google.android.gms.common.internal.zzf$zza.zzu(Unknown Source)
       at com.google.android.gms.common.internal.zzf$zze.zzxa(Unknown Source)
       at com.google.android.gms.common.internal.zzf$zzd.handleMessage(Unknown Source)
       at android.os.Handler.dispatchMessage(Handler.java:105)
       at android.os.Looper.loop(Looper.java:156)
       at android.app.ActivityThread.main(ActivityThread.java:6524)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:941)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:831)
```
Any idea about this crash?

**Comment by jkasten2** [comment]
@JustinLim0318 This was fixed in the [latest 3.5.6 version](https://github.com/OneSignal/OneSignal-Android-SDK/releases/tag/3.5.6) released on May 11th.

**Comment by JustinLim0318** [comment]
@jkasten2 alright, thanks for the response
